conf/ocsci/rosa.yaml

```diff
@@ -18,14 +18,14 @@ DEPLOYMENT:
   force_download_ocm_cli: True
   ocm_cli_version: '0.1.46'
   force_download_rosa_cli: True
-  rosa_cli_version: '1.1.2'
+  rosa_cli_version: '1.1.7'
```

Comment:
Is the version of rosa cli depend somehow on OCP or ODF version?
Like that for 4.8 we should use this and from 4.9 that?
Or this version applies for all versions?

Reply:
This version applies to all versions. From one version of rosa cli could be installed multiple OCP versions.